src/Aquifer.Public.API/Endpoints/ParentResources/List/Request.cs

```diff
@@ -0,0 +1,14 @@
+namespace Aquifer.Public.API.Endpoints.ParentResources.List;
+
+public sealed class Request
```

Comment:
I just realized that this route should probably also support paging.  We don't have a lot of collections now (only 17 in production) but there could be a larger number in the future and it's impossible to add paging later without breaking consumers.  @jwinston-bn, what do you think?  Are we likely to have more than 100 some day?

Reply:
Update: I added paging.